websites/P/Pronote/presence.ts

```diff
@@ -0,0 +1,51 @@
+const presence = new Presence({
+  clientId: "886576430408929291"
+});
+
+function get_url_extension( url ) {
+  return url.split(/[#?]/)[0].split('.').pop().trim();
+}

```

Comment:
Camel case; not snake case
```suggestion
function getUrlExtension( url ) {
  return url.split(/[#?]/)[0].split('.').pop().trim();
}

```